packages/engine/Source/Scene/ITwinData.js

```diff
@@ -205,4 +206,52 @@ ITwinData.createDataSourceForRealityDataId = async function loadRealityData(
   return KmlDataSource.load(tilesetAccessUrl);
 };
 
+/**
+ * Load data from the Geospatial Features API as GeoJSON.
+ *
+ * NOTE: this function does not implement the full WFS schema yet so the limit
+ * should be larger than the entire dataset to retrieve everything
```

Comment:
```suggestion
 * Load the first page of data from the Geospatial Features API as GeoJSON.
 *
```

Reply:
The default is already set to the max, so I don't think any action is needed by the user at this time. Further improvements should be documented in GitHub and not the code or documentation.